content/doc/book/pipeline/syntax.adoc

```diff
@@ -485,6 +479,22 @@ automatically be defined: `MYVARNAME_USR` and `MYVARNAME_PSW` respectively.
 | Inside the `pipeline` block, or within `stage` directives.
 |===
 
+===== Supported Credentials Type
+
+Secret Text:: 
+the environment variable specified will be set to the Secret Text content
+Secret File::
+the environment variable specified will be set to the location of the File
+file that is temporarily created
+Username and password:: 
+the environment variable specified will be set to `username:password` and two
+additional environment variables will be automatically defined: `MYVARNAME_USR`
+and `MYVARNAME_PSW` respectively.
+SSH with Private Key:: 
+the environment variable specified will be set to the location of the SSH key 
+file that is temporarily created and two additional environment variables may
```

Comment:
It is surprising to me that the SSH private key would ever define a username or a password environment variable.  A username might be helpful in the cases where it is needed inside the ssh destination (`ssh://username@hostname:port/path/to/repo`), but I don't see any case where a password would be available or useful.

Are you certain that there are cases where a password environment variable is set?  If so, what value is assigned to that variable?

Reply:
@MarkEWaite The password in the case of SSH Credentials is for the passphrase. Maybe I can clarify this.

Reply:
Could you call it 'passphrase' rather than 'password'?  

The [ssh-keygen man page](https://linux.die.net/man/1/ssh-keygen) calls it a `passphrase` rather than a password.  The minor word difference may help some people realize that it is the (optional) passphrase for the private key.